src/main/java/com/matt/forgehax/mods/MatrixNotifications.java

```diff
@@ -113,7 +113,7 @@
           .<String>newSettingBuilder()
           .name("skin-server-url")
           .description("URL to the skin server. If left empty then no image will be used.")
-          .defaultTo("https://visage.surgeplay.com/face/160/")
+          .defaultTo("https://minotar.net/helm/")
```

Comment:
Any specific reason for this change?

Reply:
It has a shorter url and looks nicer in chat on my super small screen :) I'll revert, pushed by mistake, was supposed to be a default change just for me